TTS/utils/samplers.py

```diff
@@ -0,0 +1,68 @@
+import torch
```

Comment:
Maybe loge it in the `trainer.torch`?

that way we can share this among projects if it is generally applicable.

Reply:
Yeah we can do it in a future PR when we merge the  FactorTrainerAPI in dev. Maybe the better is just merging this PR for now.